Sources/MapboxMaps/Snapshot/Snapshotter.swift

```diff
@@ -251,6 +257,14 @@ public class Snapshotter {
                                 bearing: bearing?.NSNumber,
                                 pitch: pitch?.NSNumber))
     }
+
+    // MARK: - Telemetry
+    internal func sendTurnstileEvent() {
+        let accessToken = ResourceOptionsManager.default.defaultAccessToken()
+        let eventsManager = EventsManager(accessToken: accessToken)
```

Comment:
We create a single EventsManager per MapView instance, but this will create a separate EventsManager per snapshot (not per Snapshotter). We could avoid paying the initialization cost repeatedly if we just created this once at Snapshotter initialization.